Banner Manager Module

sourav8256 opened the pull request:
2. Interface to see all banners in groups (Recent/Active/Inactive)
3. Interface to select the sites in which banner will be displayed and delete banners
4. Interface to set banners active/inactive
5. Interface to add new websites to the list
6. Block for fetching and displaying banners on different websites
7. Form to set variables (in settings.inc file)

sourav8256 committed:
1. New banner creation form
2. Interface to see all banners in groups (Recent/Active/Inactive)
3. Interface to select the sites in which banner will be displayed and delete banners
4. Interface to set banners active/inactive
5. Interface to add new websites to the list
6. Block for fetching and displaying banners on different websites
7. Form to set variables (in settings.inc file)

sourav8256 committed:
Enabled file upload to local directory in new_banner form

sourav8256 committed:
Displaying database error message

sourav8256 committed:
last_updated field added in .install

sourav8256 committed:
Renamed the module form banner_manager to fossee_site_banner

Sashi20 reviewed (changes requested):
Remove the file path which is hard coded from all the files. And use the file path provided in the settings. Also, add a link to add the new domain names when there are none.

sourav8256 committed:
1. Fixed hard coded url in javascript
2. New banner link in banners page
3. Ajax form to add multiple website
4. Table for available websites in add-website page with option to delete them
5. Changed _ to - in urls
6. Automatically creates banner directory if not available

sourav8256 committed:
Removed call to hook_validate() when adding field with ajax

sourav8256 committed:
Fixed small error in banners page

sourav8256 committed:
Fixed displaying of broken image when no banner is selected

sourav8256 committed:
1. Form to edit websites
2. Form to edit banners
3. Removed hard coded urls
4. File extension in banners settings are seperated by commas
5. banner_dir variable in block is fetched from database

sourav8256 committed:
File upload field added to edit form page

sourav8256 committed:
1. option to update database tables from settings page
2. partial progress in email about banner status

sourav8256 committed:
added sending of email to the banner administrator on creating new banners or when banner has less than 7 days of validity
